src/scandeval/benchmark_config_factory.py

```diff
@@ -156,8 +156,8 @@ def get_correct_language_codes(language_codes: str | list[str]) -> list[str]:
 
 
 def prepare_languages(
-    language: str | list[str] | None,
-    language_codes: list[str],
+    language_codes: str | list[str] | None,
+    default_language_codes: list[str],
 ) -> list[Language]:
     """Prepare language(s) for benchmarking.
```

Comment:
Docstring also needs updating with the new variable names

Reply:
You can add this as a check using ruff if you would like.